AnimalKingdom/src/animalKingdom/Animal.java

```diff
@@ -0,0 +1,36 @@
+package animalKingdom;
+
+public abstract class Animal {
+
+    String name;
+    int Id;
```

Comment:
For encapsulation, it is good practice in Java to have all class variables use the `private`modifier (or `protected` for those that you will use in subclasses).